doc/src/learn/exchange-integration-guide.md

```diff
@@ -0,0 +1,484 @@
+
+   The benefits of this method include: no gas fees for merging or splitting tokens, and the abstracted token merge and split. The `sui_paySui` operation is a native function, so the merge and split operations are not considered Sui transactions. The gas fees for them match typical transactions on Sui.You can use this operation to split coins in your own address by setting the recipient as your own address. Note that the total value of the input coins must be greater than the total value of the amounts to send.
+
+ * [sui_pay](https://docs.sui.io/sui-jsonrpc#sui_pay)
+   This method is similar to sui_paySui, but it accepts any kind of coin or token instead of only SUI. You must include a gas token, and all of the coins or tokens must be the same type.
+
+ * [sui_transferSui](https://docs.sui.io/sui-jsonrpc#sui_transferSui)
+    This method accepts only one SUI token object and an amount to send to the recipient. It uses the same token for gas fees, so the amount to transfer must be strictly less than the value of the SUI token used.
+
+## SUI Staking and Delegation
+
+The Sui blockchain uses a delegated Proof-of-Stake mechanism (DPoS). This allows SUI token holders to delegate their tokens to any validator of their choice. When someone delegates their SUI tokens, it means those tokens are locked for the entire epoch. Users can withdraw their stake and stake with a different validator between epochs.
+
+SUI holders who delegate their tokens to validators earn rewards for helping secure the Sui network. Sui determines rewards for delegation based on stake rewards on the network, and distributes them at the end of each epoch.
+
+The total voting power in the Sui Network is always 10,000. The voting power of each individual validator is similar to basis points. For example, a voting power of 101 = 1.01%. Sui's quorum threshold (number of votes needed to confirm a transaction) is 6,667 (which is greater than 2/3). The voting power for a single validator is capped at 1,000 (10%) regardless of how much stake the validator has.
+
+## Delegated staking functions
+
+Sui supports the following API operations related to staking and delegation. You can find the source code in the [sui_system](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/governance/sui_system.move) module.
+
+ * `request_add_delegation`
+ Add delegated stake to a validator's staking pool.
+
+```rust
+public entry fun request_add_delegation(
+   self: &mut SuiSystemState,
+   delegate_stake: Coin<SUI>,
+   validator_address: address,
+   ctx: &mut TxContext,
+) {
+   validator_set::request_add_delegation(
+       &mut self.validators,
+       validator_address,
+       coin::into_balance(delegate_stake),
+       option::none(),
+       ctx,
+   );
+}
```

Comment:
Not a blocker, but I don't think the bodies of these functions are needed here (and below), and just the signature would be OK so we may want to consider removing the function bodies.

Reply:
@patrickkuo